<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Passengers</class>
 <widget class="QWidget" name="Passengers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пассажиры</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_add">
     <property name="text">
      <string>Добавить пассажира</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_delete">
     <property name="text">
      <string>Удалить пассажира</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="checkBox_search">
     <property name="text">
      <string>Отфильтровать данные</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_change">
     <property name="text">
      <string>Изменить пассажира</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Номер паспорта</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_number"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Дата рождения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QDateEdit" name="dateEdit_date"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
